python_modules/dagster-graphql/dagster_graphql/schema/schedules/schedules.py

```diff
@@ -110,6 +102,19 @@ def resolve_futureTicks(self, _graphene_info, **kwargs):
     def resolve_futureTick(self, _graphene_info, tick_timestamp):
         return GrapheneFutureInstigationTick(self._schedule_state, float(tick_timestamp))
 
+    @staticmethod
+    def with_default_state(instance, external_schedule, stored_state):
```

Comment:
Hm I think you addressed my previous feedback about `fetched_state=False` being an awkward construction in another way that is also fine with me (by always passing in the stored_state to the constructor of the GrapheneSchedule). But once that change has been made and all callsites to GrapheneSchedule are passing in the stored_state, I don't see why this logic can't be in the GrapheneSchedule constructor and applied every time. (I.e. i don't think you need the static method anymore, you have rendered it not needed by passing in stored_state everywhere)